USING-CONFLUENCE.md

```diff
@@ -0,0 +1,69 @@
+# Using Confluence
+
+The confluence command uses the [Web API Confluence Dashboard](http://web-confluence.appspot.com/#!/) to generate new or improved browser compatibility data ready for submission in pull requests. The Web API Confluence Dashboard provides metrics and support data about web platform features across multiple browsers. The MDN Browser Compatibility Database provides a command for using Confluence to generate new and updated JSON files. This page explains how to do this.
+
+## Prerequisites
+
+Before using the confluence command make sure you have done all of the items below.
+
+* On GitHub fork this repo.
+* In this repo, add your fork as a remote repo: `git remote add fork-name path-to-fork`
```

Comment:
OK, a non-copy thing: in my experience, a lot of people have a really hard time with multiple remotes, so maybe let's suggest cloning the fork and calling it a day?

```suggestion
* Clone your fork.
```

Reply:
No. This tool can only be used locally. Since they can't push directly to origin, they need multiple repos.

Reply:
If I can't push to mdn/browser-compat-data directly, then I **must** have a fork in order to open a PR.

Reply:
OK, I don't think I'm any closer to understanding this, but no reason to hold things up over one line. Resolving.